donut/static/js/groups/positions.js

```diff
@@ -26,6 +26,7 @@ function populateAdminGroups(approvedGroupIds, approvedGroupNames) {
         $('#groupCreate option:last').after(newOption);
         $('#groupDel option:last').after(newOption);
         $('#groupPosHold option:last').after(newOption);
+        $('#groupDelPosHold option:last').after(newOption);
```

Comment:
Hmm, maybe it's too late to change this style now; but it would probably be faster to select the all the `<select>`s and just call `.append()`, rather than looking for the last `option` in each `select`. Could do something like this, where the class `group-select` is added to all the relevant `<select>`s:
```js
$('.group-select').append(newOption)
```

Reply:
I think this makes sense, maybe I can put this in particular as a TODO and have some of the new members look into as a starter task?

Reply:
Sure, sounds reasonable